### PR TITLE
7900-In single post view deleting a comment updates the counter

### PR DESCRIPTION
**Pull request by tclaus**
Added removeComment event in interactions to update the comments-counter after remove

### CODE REVIEW

File: app/assets/javascripts/app/models/post/interactions.js [hunk 1]
```diff
@@ -85,6 +85,10 @@ app.models.Post.Interactions = Backbone.Model.extend({
     });
   },
 
+  removedComment: function() {
+      this.set({"comments_count": this.get("comments_count") - 1});
```
Comment (diaspora-code-review) [inline app/assets/javascripts/app/models/post/interactions.js]: Expected indentation of 4 space characters but found 6.

File: app/assets/javascripts/app/views/comment_stream_view.js [hunk 2]
```diff
@@ -135,7 +135,10 @@ app.views.CommentStream = app.views.Base.extend({
   },
 
   removeComment: function(comment) {
-    this.$("#" + comment.get("guid")).closest(".comment.media").remove();
+    var result = this.$("#" + comment.get("guid")).closest(".comment.media").remove();
+    if (result.hasClass('deleting') ) {
```
Comment (diaspora-code-review) [inline app/assets/javascripts/app/views/comment_stream_view.js]: - Strings must use doublequote.
- There should be no spaces inside this paren.

**Comment by tclaus** [comment]
BTW: Solves Issue #7938

**Comment by HankG** [comment]
I've pulled this down and confirm that this resolves the bug in #7900 .  I believe this is ready to be merged.

**Review by HankG — APPROVED** [review]

**Review by Flaburgan — CHANGES_REQUESTED** [review]
Sorry for the late review, and thank you for your work!
I tested your code and it works well. It would be awesome however if you could [write a test](https://wiki.diasporafoundation.org/Testing_workflow) for it. A Jasmine one could be enough, add it in `/spec/javascripts/app/views/single-post-view/single_post_interaction_counts_spec.js`.

You can then squash your commits (with `git fetch upstream`, `git rebase -i upstream/develop` then replacing `pick` by `f` for the commits below the first one) and you'll be ready to go!

**Comment by tclaus** [comment]
@Flaburgan Sorry - I checked the Jasmine tests but I have no clue how to set a comment, get the current number displayed and then remove a comment and check the displayed number again. Can someone take over the tests part or give me a deeper advice?

**Comment by ThorstenClaus** [comment]
@denschub Any progress here?

**Comment by tclaus** [comment]
@denschub Mention for reviewing this. Base code has changed over time - is this still a valid PR or can this be closed now?

**Comment by denschub** [comment]
Sorry. I wanted to write a test, but then never did. I'm also not really in the mood today, so.. let's just merge this without tests.

Merged as https://github.com/diaspora/diaspora/commit/d0eb711ca0ac1541756d2f25d963095a3236db11.

**Pull request closed without merging**